pootle/apps/pootle_store/models.py

```diff
@@ -872,10 +872,7 @@ def toggle_qualitycheck(self, check_id, false_positive, user):
                          unit=self, store=self.store, type=sub_type,
                          quality_check=check)
         sub.save()
-
-        # update timestamp
-        # log user action
-        self.save()
+        self.save(revision=Revision.incr())
```

Comment:
not 100% - but that was what i figured the effect of having those other vars set was